benchmarking/example_classification.py

```diff
@@ -54,76 +71,88 @@
     X_test, y_test = X[test_idx], y[test_idx]
     X_valid, y_valid = X[valid_idx], y[valid_idx]
 
-    records = []
+    X_train_mols = [MolFromSmiles(smiles) for smiles in X_train]
+    X_valid_mols = [MolFromSmiles(smiles) for smiles in X_valid]
+    X_test_mols = [MolFromSmiles(smiles) for smiles in X_test]
+
+    try:
+        conf_gen = ConformerGenerator()
+        X_train_conf = conf_gen.transform(X_train_mols)
+        X_valid_conf = conf_gen.transform(X_valid_mols)
+        X_test_conf = conf_gen.transform(X_test_mols)
+        conformers_generated = True
+    except ValueError as e:
+        conformers_generated = False
+        print(f" - Error: {e}")
```

Comment:
If we can't generate conformers, we should probably just raise an error and stop

Reply:
how about using ConformerGenerator.transform_x_y ?